init/install_services.sh

```diff
@@ -53,6 +53,7 @@ kubectl create secret generic openai-secrets \
 
 helm upgrade --install weather-agent oci://ghcr.io/lmos-ai/weather-agent-chart --version 1.0.7-SNAPSHOT
 helm upgrade --install news-agent oci://ghcr.io/lmos-ai/news-agent-chart --version 1.0.7-SNAPSHOT
+helm upgrade --install arc-view-runtime-web oci://ghcr.io/lmos-ai/arc-view-runtime-web-chart --version 0.1.0
```

Comment:
I wouldn't add this to the "#Install agents" section, but rather create a new separate section for it.